fatek/symbol.py

```diff
@@ -74,7 +74,8 @@ def __calculate_offset(self):
         offset = self.offset_dict[register]
         if register == 'C' and current_value and number >= 200:
             number %= 200
-            offset = 9700 + 2 * (number)
+            number = number + number
+            offset = 9700
```

Comment:
Yes, C255 should be 9810.

The spec is in the page 13-45 of [offical doc](http://www.fatek.com/tw/data%2Fftp%2FPLC%2FFBs_Manual%2FManual_2-Cht%2FChapter_13.pdf). Its addresses were shifted 1.